### PR TITLE
CDPS-774: Height edit routes + controller

### PR DESCRIPTION
**Pull request by danielburnley**

### CODE REVIEW

File: server/views/pages/edit/singleField.njk [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{% extends "../index.njk" %}
```
Comment (brightonsbox) [inline server/views/pages/edit/singleField.njk:1]: Is this in use anywhere?

**Review by brightonsbox — APPROVED** [review]
LGTM

File: server/views/pages/edit/singleField.njk (hunk 1 above)
Comment (danielburnley) [reply server/views/pages/edit/singleField.njk:1]: Not at the moment - it was used by the height page with the intent to be used by the weight page until I got the designs and needed the switcher, figured I'd keep it in for now at least 😅

**Review by brightonsbox — APPROVED** [review]
LGTM

**Pull request merged**